### DISCUSSION TITLE
Option to Place on Right Side of Screen

### DISCUSSION THREAD

**Discussion by aqres** [lawand-dot-io/taskbar#91] [Feature requests]
As the title says, the option to place the bar on the right side of the display

**Comment by lawand-dot-io** [comment]
Hello. This is a duplicate of https://github.com/lawand-dot-io/taskbar/discussions/9 

Feel free to upvote that request to increase the chance it will be implemented. Thanks and hope you find Taskbar useful

**Reply by aqres** [reply]
Ah sorry

**Reply by lawand-dot-io** [reply]
No, all good :)
